package.xml

```diff
@@ -38,8 +38,6 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
   <depend>atlas</depend>
   <depend>eigen</depend>
   <depend>gfortran</depend>
```

Comment:
are these dependencies needed ? they don;t seem to be part of the turtlebot dependencies on CI